doc/man7/EVP_MAC_POLY1305.pod

```diff
@@ -22,6 +22,21 @@ The supported controls are:
 
 =item B<EVP_MAC_CTRL_SET_KEY>
 
+EVP_MAC_ctrl_str() takes to type strings for this control:
```

Comment:
`to` should be `two` (and yeah I know, that was my misspelling...  will fix)

Reply:
I was also wondering this when I copied this sentence. And later I found `take to` is also a valid phrase in English so I thought you meant to this...